_posts/2022-03-10-introducing-openstreetmap-places.md

```diff
@@ -0,0 +1,94 @@
+
+![OSM Handshake]({% link images/osm_handshake.png %} "OSM Handshake"){: .flex .flex--center}
+
+### How does it differ from other SGID data?
+{: .text-left}
+
+- It's **not** authoritative - due to the nature of crowdsourced data, we can't guarantee that it's perfect or accurate
+- It has a different license - instead of our usual [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/) license, it carries the [ODbL 1.0](https://opendatacommons.org/licenses/odbl/summary/) license
+- It's more dynamic - it gets updated frequently, is constantly growing, and anyone can contribute to it
+
+> OpenStreetMap® is _open data_, licensed under the [Open Data Commons Open Database License](https://opendatacommons.org/licenses/odbl/) (ODbL) by the [OpenStreetMap Foundation](https://osmfoundation.org/) (OSMF).
+
+### How is it similar to other SGID data?
+{: .text-left}
+
+- It may include locations that are represented in other, authoritative, SGID datasets
+   - Examples: schools, parks, liquor stores, libraries, fire stations, etc.
+- It's available in a variety of ways and data formats
+   - ArcGIS Online services
+   - OpenData
+   - OpenSGID - our Postgres database in the Google Cloud
+   - Downloads in your favorite data format (shapefile, file geodatabase, GeoJSON, KML, CSV)
+
+
+### What else should I know?
+{: .text-left}
+
+- The OSM data isn't perfect, so you may find some typos, missing locations, locations that no longer exist, etc.
+- OSM data tends to have a European 'flavour' (see what I did there?).  Be mindful that many categories or attributes may be spelled in a British/European English manner.  Business or location names shouldn't be affected, but attribute examples you might encounter include:
+   - centre (instead of center, ex: sports_centre, arts_centre, garden_centre)
+   - theatre (instead of theater)
+   - caravan_site (instead of rv_park or trailer_park
+   - archaeological (instead of archeological)
+   - chemist (instead of pharmacy)
+   - tyres (instead of tires)
+- There are two types of addresses provided in the data:
+   1. The address contained in the OSM data (`osm_addr`).  These were input by OSM contributors, but only a small percentage of the points come with an address (~24%).
+   1. The address of the nearest UGRC address point (`ugrc_addr`).  These are not official addresses of the businesses or points of interest, but provide a very close address for a greater percentage of points (~59%).  They are the nearest address (within 25m) in the [UGRC Address Points](https://opendata.gis.utah.gov/datasets/utah-address-points/explore) dataset.  The `addr_dist` field provides the distance (in meters) from the OSM point to the nearby address point.
```

Comment:
**thought**: should we link internally to the address points?

Reply:
Yeah, I was thinking about that too.